### ISSUE TITLE
Inspector Tabs: Insufficient color contrast

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#7575]
## Bug Description

WCAG AA requirement for color contrast is a minimum of 4.5.

The inspector tabs (Design / Document / Checklist) do not meet this requirement, with a contrast ratio of only 3.53.

This is due to `theme.colors.fg.tertiary` using `gray[40]` (`#767E80`), whereas a lighter gray would be needed.

https://github.com/google/web-stories-wp/blob/4c891bd2b6b10815c72b06d7bbccd15718da79b1/assets/src/edit-story/components/tabview/index.js#L72-L73

## Expected Behaviour

Color contrast is increased and meets a11y requirements

## Screenshots

Current situation:

![Screenshot 2021-05-18 at 23 26 31](https://user-images.githubusercontent.com/841956/118725879-cf40bf00-b830-11eb-84d1-1a7f057d9178.png)

Using `brand.gray[40]` (`#767E80`), color contrast ratio is only 3.53 ❌ 

Suggested change:

![Screenshot 2021-05-18 at 23 26 44](https://user-images.githubusercontent.com/841956/118725756-aa4c4c00-b830-11eb-944e-2850aba99b7c.png)

Using `brand.gray[30]` (`#919899`), color contrast ratio increases to 4.99 ✅ 

## Additional Context

- Plugin Version: 1.7
- WordPress Version: 5.7
- Operating System: macOS
- Browser: Chrome

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
@aaecheverria Is the suggested color above OK or would you suggest a different gray?

**Comment by ivayloklissarov** [comment]
Looks good @swissspidy , I've updated the designs system with this.

**Comment by swissspidy** [comment]
Awesome, thanks.

Assigning to @miina for prioritization.
